code-challenges.test.js

```diff
@@ -54,3 +116,10 @@ const numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+
+// const addNumsArray = numbersToAdd1 => numbersToAdd1.map((sum => value => sum += value)(0))
+// output: [2, 6, 51, 60]
+// const addNumsArray = numbersToAdd2 => numbersToAdd1.map((sum => value => sum += value)(0))
+// output: [0, 7, -1, 11]
+// const addNumsArray = numbersToAdd3 => numbersToAdd1.map((sum => value => sum += value)(0))
```

Comment:
Remember proper functions are **"reusable"** so it should only be declared once!